app/client/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:16
+
+WORKDIR /
+COPY ./package.json ./
+RUN npm install
+
+
+COPY . ./
```

Comment:
Better to copy into a directory above the root, then set the workdir there (drop the `WORKDIR` from above)

```
COPY  ./package.json /app/package.json
WORKDIR /app
```

then

```
COPY . /app
```